apps/intellij/src/main/kotlin/dev/nx/console/run/runGeneralCommand.kt

```diff
@@ -0,0 +1,64 @@
+package dev.nx.console.run
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.executors.DefaultRunExecutor
+import com.intellij.execution.filters.TextConsoleBuilderFactory
+import com.intellij.execution.process.KillableColoredProcessHandler
+import com.intellij.execution.ui.RunContentDescriptor
+import com.intellij.execution.ui.RunContentManager
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import dev.nx.console.NxIcons
+import dev.nx.console.nxls.NxlsService
+import dev.nx.console.utils.ProjectLevelCoroutineHolderService
+import dev.nx.console.utils.nxBasePath
+import java.io.File
+import javax.swing.SwingUtilities
+import kotlinx.coroutines.launch
+
+fun runGeneralCommand(project: Project, projectName: String, helpCommand: String) {
```

Comment:
It works fine like this but we could also think about making this a `NxRunHelpService` or something like that. It's really simple with a `@Service(Service.Level.PROJECT)` annotation. [Read more here](https://plugins.jetbrains.com/docs/intellij/plugin-services.html#light-services).
In that case we would automatically get our own coroutine scope instead of having to use the `ProjectLevelCoroutineHolderService` (which would make debugging easier in case anything goes wrong). Also, we wouldn't have to pass `project` to the function - instead to the service's `getInstance` method though. 
So it doesn't make the biggest of differences but it seems to be the slightly more "intellij-y" way of doing things.